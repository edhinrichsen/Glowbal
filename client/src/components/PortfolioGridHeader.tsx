import React from 'react';
import { PageHeader } from 'antd';

type PortfolioGridHeaderProps = {
  title: React.ReactNode;
  socialLinks?: JSX.Element;
};

function PortfolioGridHeader(props: PortfolioGridHeaderProps) {
  return <PageHeader title={props.title} extra={props.socialLinks} />;
}

export default PortfolioGridHeader;
