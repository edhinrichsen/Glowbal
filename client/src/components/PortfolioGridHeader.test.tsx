import React from 'react';
import { shallow } from 'enzyme';
import PortfolioGridHeader from './PortfolioGridHeader';

describe('Testing PortfolioGridHeader', () => {
  it('renders without crashing', () => {
    const component = shallow(<PortfolioGridHeader title="Title" />);
    expect(component).toMatchSnapshot();
    component.unmount();
  });
});
