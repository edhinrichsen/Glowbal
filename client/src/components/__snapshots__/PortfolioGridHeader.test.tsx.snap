// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing PortfolioGridHeader renders without crashing 1`] = `
<PageHeader
  title="Title"
/>
`;
